src/sentry/issues/occurrence_consumer.py

```diff
@@ -54,23 +54,26 @@ def create_rate_limit_key(fingerprint: str) -> str:
 def is_rate_limited(
     fingerprint: str,
 ) -> bool:
-    rate_limit_enabled = options.get("issues.occurrence-consumer.rate-limit.enabled")
-    if not rate_limit_enabled:
+    try:
+        rate_limit_enabled = options.get("issues.occurrence-consumer.rate-limit.enabled")
+        if not rate_limit_enabled:
+            return False
+
+        rate_limit_key = create_rate_limit_key(fingerprint)
+        rate_limit_quota = Quota(**options.get("issues.occurrence-consumer.rate-limit.quota"))
+        granted_quota = rate_limiter.check_and_use_quotas(
+            [
+                RequestedQuota(
+                    rate_limit_key,
+                    1,
+                    [rate_limit_quota],
+                )
+            ]
+        )
+        return not granted_quota.granted
+    except Exception:
```

Comment:
I would also add in a `logger.exception("Failed to check issue platform rate limiter")` here so that we don't just swallow the problem and not realise